content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
-          For example, the patterns include keyboard interaction models that combine a normative requirement specified by ARIA with conventions established by popular operating systems and software.
-          In the section on <a class="specref" href="#managingfocus">Managing focus</a>, ARIA includes normative language stating that the <kbd>Tab</kbd> key should not move focus inside of certain types of widgets.
-          However, ARIA does not specify how specific keys should move focus inside of those widgets.
-          So, the APG Task Force relies on broadly utilized keyboard conventions to provide each pattern with a list of keyboard commands for navigation and operation.
-          While those lists are not normative, the majority of such conventions are so well established that it is advisable to treat them as requirements.
-          Where that is not the case, the guidance is clearly marked as optional.
-          Note that there are some keyboard commands where users could be better served if a web widget were to dynamically adjust to mirror the users' operating system conventions.
-          It is a <a href="https://github.com/w3c/aria-practices/issues/1438">known issue</a> that some APG keyboard interaction models do not yet provide sufficient macOS-specific guidance.        </p>
+          APGのガイダンスの一部は規範的な仕様によってカバーされていません。
+          例えば、パターンには、ARIAによって指定された規範的な要件と、一般的なオペレーティングシステムおよびソフトウェアによって確立された規約を組み合わせたキーボードインタラクションモデルが含まれています。
+          <a class="specref" href="#managingfocus">フォーカスの管理</a>のセクションでは、ARIAには特定のタイプのウィジェット内で<kbd>Tab</kbd>キーがフォーカスを移動させないようにするという規範的な言語が含まれています。
+          しかし、ARIAは、これらのウィジェット内で特定のキーがどのようにフォーカスを移動させるべきかを指定していません。
+          そのため、APGタスクフォースは、広く利用されているキーボードの規約に依存して、ナビゲーションおよび操作のためのキーボードコマンドのリストを各パターンに提供します。
+          これらのリストは規範的ではありませんが、そのような規約の大部分は非常によく確立されているため、要件として扱うことが望ましいです。
+          その場合でない場合、ガイダンスは明確にオプショナルとしてマークされています。
+          注記として、ウェブウィジェットがユーザーのオペレーティングシステムの規約に動的に適応するように調整された場合、ユーザーにとってより良いサービスが提供される可能性があるキーボードコマンドがいくつかあります。
+          一部のAPGキーボードインタラクションモデルがmacOS固有のガイダンスを十分に提供していないというのは<a href="https://github.com/w3c/aria-practices/issues/1438">既知の問題</a>です。
+        </p>
       </section>
 
       <section id="caution-when-deviating">
-        <h2>Exercise Caution If Deviating from APG Patterns</h2>
+        <h2>APGパターンから逸脱する場合の注意</h2>
         <p>
-          While APG is not a standard, its patterns and practices do convey requirements from normative standards.
-          To help authors address a wide variety of UI design needs, the patterns and practices describe ranges of possibilities and options that satisfy those standards.
-          The examples each represent a specific way of implementing one or more patterns; they are intended to be illustrative, not prescriptive.
+          APGは標準ではありませんが、そのパターンと実践は規範的な標準からの要件を伝えます。
+          著者がさまざまなUIデザインのニーズに対処するのを助けるために、パターンと実践はそれらの標準を満たす可能性のある範囲とオプションを説明します。
```

Comment:
WCAGではauthor -> コンテンツ制作者としているので、倣いました

```suggestion
          コンテンツ制作者がさまざまなUIデザインのニーズに対処するのを助けるために、パターンと実践はそれらの標準を満たす可能性のある範囲とオプションを説明します。
```